nomenclature/processor/iamc.py

```diff
@@ -14,7 +14,7 @@ class IamcDataFilter(BaseModel):
     unit: List[str] | None = None
     year: List[int] | None = None
 
-    @field_validator("*", mode="before")
+    @field_validator(*IAMC_IDX + ["year"], mode="before")
```

Comment:
You could've still kept using `"*"`, no? 
Might be easier to understand that this validator is supposed to be applied to all fields for someone who's not familiar with `IAMC_IDX`

Reply:
This doesn't work because it would also be applied to the `upper_bound` and 'lower_bound` attributes of the class that inherits from the IamcDataFilter...

Reply:
Ah, you're right, never mind.